.github/workflows/main.yml

```diff
@@ -0,0 +1,34 @@
+name: Build and Deploy
+on:
+  pull_request:
```

Comment:
Let's make it run only when `push to master`.

Reply:
DONE